subiquity/controllers/cmdlist.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2019 Canonical, Ltd.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import asyncio
+
+from subiquity.controller import NoUIController
+
+
+class CmdListController(NoUIController):
+
+    autoinstall_default = []
+    cmds = ()
```

Comment:
Should this be a list instead of tuple?

Reply:
perhaps autoinstall_default should be ()? I don't think it matters a great deal although having them different is indeed odd.